packages/core/application/core-application-browser-internal/src/ui/app_container.tsx

```diff
@@ -129,11 +128,29 @@ export const AppContainer: FC<Props> = ({
 };
 
 const AppLoadingPlaceholder: FC<{ showPlainSpinner: boolean }> = ({ showPlainSpinner }) => {
+  const { euiTheme } = useEuiTheme();
+  const appContainerFadeIn = keyframes({
+    '0%': { opacity: 0 },
+    '50%': { opacity: 0 },
+    '100%': { opacity: 1 },
+  });
+  const appContainerStyles = css({
+    position: 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    zIndex: euiTheme.levels.header,
```

Comment:
See: https://github.com/elastic/eui/discussions/6828

> ```
> $euiZLevel1 | This can be either levels.header, levels.flyout, or levels.maskBelowHeader, depending on the context/use case.
> ```



```

Reply:
That global banners live at the same level of the header makes this a reasonable selection. Tested locally, and the banner appears as expected.